app/assets/stylesheets/application.css.erb

```diff
@@ -0,0 +1,124 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+/* Base Design ------------------------------------------------ */
+
+html {
+  font-family: 'Nixie One', cursive;
+  background: #fff url(<%= asset_path 'strawberries.jpg' %>) no-repeat center fixed;
+  background-size: cover;
+}
+
+body {
+  -webkit-border-radius: 20;
+  -moz-border-radius: 20;
+  border-radius: 20px;
+  margin: 20px auto;
+  width: 960px;
+  background-color: rgba(255, 255, 255, 0.8);
+  padding: 20px;
+}
+
+/* Homepage --------------------------------------------------- */
+
+.homepage {
+  text-align: center;
+}
+
+.homepage h1 {
+  text-decoration: underline;
+  font-family: 'Nixie One', cursive;
+  font-size: 2em;
+}
+
+.homepage h2 {
+  margin: 0 auto;
+  padding: 20px 0;
+  width: 450px;
+  font-size: 1.1em;
+}
+
+.homepage .button_link, .submit_button {
+  display: inline-block;
+  -webkit-border-radius: 10;
+  -moz-border-radius: 10;
+  border-radius: 10px;
+  border: 2px solid #FFF;
+  color: #FFF;
+  background: #000;
+  padding: 10px 15px;
+  text-decoration: none;
+}
+
+/* Tables ----------------------------------------------------- */
+
+th {
+  text-align: left;
+  font-size: 1.2em;
+  text-decoration: underline;
+  padding: 5px;
+}
+
+td {
+  text-align: left;
+  padding: 5px;
+}
+
+/* Links ------------------------------------------------------ */
+
+a:link,
+a:visited {
+  color: #A00;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #000;
+  font-style: italic;
+}
+
+/* Forms ------------------------------------------------------ */
+
+.form_field {
+  display: block;
+  margin-right: 25px;
+  padding-bottom: 3px;
+  margin-bottom: 15px;
+  width: 175px;
+  height: 13px;
+}
+
+.submit_button {
+  font-family: 'Nixie One', cursive;
+  font-size: 13px;
+}
+
+.submit_button:hover {
+  font-style: italic;
+}
+
+/* Footer ---------------------------------------------------- */
+
+.home_link {
```

Comment:
CSS naming conventions say that '-' should be used instead of '_' to separate words.